src/equations/equations.jl

```diff
@@ -77,6 +77,7 @@ function cons2prim(u, ::AbstractEquations) end
 # FIXME: Deprecations introduced in v0.3
 @deprecate varnames_cons(equations) varnames(cons2cons, equations)
 @deprecate varnames_prim(equations) varnames(cons2prim, equations)
+@deprecate flux_upwind(u_ll, u_rr, orientation, equations) flux_godunov(u_ll, u_rr, orientation, equations)
```

Comment:
I assume this change has been approved by @gregorgassner and @andrewwinters5000? I vaguely recall a discussion, but don''t remember if it was just the two of us. From my side it is ok.

Reply:
@trixi-framework/principal-developers should have a look but we discussed this some time ago in a meeting (IIRC).

Reply:
I am okay with the renaming

Reply:
Ok to me as well.